src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -1,10 +1,20 @@
-import React, { useEffect, useState } from 'react';
+// @ts-check
+/* eslint-disable react/prop-types */
```

Comment:
We can clean this
```suggestion
```

Reply:
Updated https://github.com/open-craft/frontend-app-course-authoring/pull/38/commits/d37ffbf4000b14cfe5ea3065056e7c3d1da1f4e5